index.html

```diff
@@ -6,170 +6,213 @@
+               <ul class="meanmenu">
+                    <li><a href="#"><img src=" " alt="">Portoflilyo</a></li>
+                    <li><a href="#"><img src=" " alt="">About</a></li>
+                    <li><a href="#"><img src=" " alt="">Contact</a></li>
+                </ul>
+            </div>
+        </nav>
     </header>
-     <div class="cardBack01">
-        <div>
-            <h1 class="main-header"> I'm Mihertab Glad to see you!</h1>
-            <p class="section1"> I’m a software developer! I can help you build a product , 
-                feature or website Look through some of my work and experience! If you like what 
-                you see and have a project you need coded, don’t hestiate to contact me. </p>
-        </div>
-        <div class = "social-section">
-                    <h4 class="connectmywork"> LET’S CONNECT</h4>
+    <div class="cardBackmain">
+        
+            <div>
+                <h1 class="main-header"> I'm Mihertab Glad to see you!</h1>
+                <p class="section1"> I’m a software developer! I can help you build a product , 
+                    feature or website Look through some of my work and experience! If you like what 
+                    you see and have a project you need coded, don’t hestiate to contact me. </p>
+            </div>
+            <div class = "social-section">
+                        <h4 class="connectmywork"> LET’S CONNECT</h4>
                 <ul class="social-icons-mywork">
-                    <li><a href="#"><img src="./image/Vectortweet.png" alt=""></a></li>
-                    <li><a href="#"><img src="./image/VectorM.jpg" alt=""></a></li>
-                    <li><a href="#"><img src="./image/Vector5.png" alt=""></a></li>
-                    <li><a href="#"><img src="./image/VectorGit.png" alt=""></a></li>
-                    <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
+                        <li><a href="#"><img src="./image/Vectortweet.png" alt=""></a></li>
+                        <li><a href="#"><img src="./image/VectorM.jpg" alt=""></a></li>
+                        <li><a href="#"><img src="./image/Vector5.png" alt=""></a></li>
+                        <li><a href="#"><img src="./image/VectorGit.png" alt=""></a></li>
+                        <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
                 </ul>
-        </div>
-        
-      </div>
-      </section>
-
-<section class="Workssection">
-    <div class="cardBack">
-    <a class="card1" ></a>
-            <h3  class="tonic" >Tonic</h3>
-         <div>
+            </div>
+    </div>
+</section>
+<section class="Workssection" id="section-02">
+    <div class="cardBack-02" id="project-02">
+            <img class="Mob02" src="./image/Snapshoot Portfolio1 (1).png" alt="">
+             <img class="Desktop02" src="./image/Desktop01.png" alt="">
+    <div>
+        <h3  class="tonic" >Tonic</h3>
+        <div>
             <ul class="works">
```

Comment:
You did a nice job working on your project section! 💯  However, there are still some things you can improve:

- [ ] Some of your project cards are causing your page to have horizontal overlapping. Please consider fixing this! 🔧 
💡 _Hint: this is mostly caused because you're using some fixed margin and padding values._

![ezgif com-gif-maker (2)](https://user-images.githubusercontent.com/77069011/201487193-4d47f399-4c27-4d5e-92f0-2847f0fafd27.gif)

- [ ] Please, consider following as close as possible the given Figma design 🎨 , at the moment the cards don't have the correct size and are missing the action buttons. 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/201487348-103d6d4b-7191-4811-99d8-388158b04978.png) | ![image](https://user-images.githubusercontent.com/77069011/201487338-adce7f0f-2659-485b-a85a-a1a2b6b533d5.png) |

Please, consider applying these changes! 🤓 🛠️